scvi/data/_manager.py

```diff
@@ -224,6 +267,20 @@ def validate(self) -> None:
             adata, self.adata = self.adata, None  # Reset self.adata.
             self.register_fields(adata, self._source_registry, **self._transfer_kwargs)
 
+    def update_setup_method_args(self, setup_method_args: dict):
+        """
+        Update setup method args.
+
+        Parameters
+        ----------
+        setup_method_args
+            This is a bit of a misnomer, this is a dict representing kwargs
+            of the setup method that will be used to update the existing values
+            in the registry of this instance.
+        """
+        if setup_method_args is not None:
```

Comment:
nit: do we ever expect to call this with a None?

Reply:
No whoops, should've removed this